MRN-Code/coinstac
# Section Lists and Column Headings

* **Problem:**
   1. Missing headings on pipeline step covariates are confusing.
   1. Users might want quicker access to objects they own and find that difficult if object lists are too long.
* **Solution:**
   1. Added headings to pipeline step covariate columns.
   1. Object list pages sectioned off to highlight owned objects if enough objects present.
* **Tests:**
   1. Add computations, pipelines, consortia in excess of of five to see lists sectioned off.
   1. View test Pipeline and expand any computation to see column headings above covariates.
